INSTALL.md

```diff
@@ -49,198 +31,135 @@ To update an existing clone you can use the following commands:
+* libxcursor (MIT): https://xorg.freedesktop.org
+* pixman (MIT): http://www.pixman.org
+* tinyxml (Zlib): http://www.grinninglizard.com/tinyxml
+* zlib (Zlib): https://www.zlib.net
 
-1. Allegro back-end (Windows, Linux): You will not need any extra
-   library because the repository already contains a modified version
-   of the Allegro library. This back-end is deprecated and will be
-   removed in future versions. All new development is being done in
-   the new Skia back-end.
+### Linux dependencies
 
-2. Skia back-end (Windows, Mac OS X): You will need a compiled version
-   of the Skia library. Please check the details about
-   [how to build Skia](#building-skia-dependency) on your platform.
+To install the required dependencies on Debian/Ubuntu, run:
 
-## Windows dependencies
+    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+
+### Windows dependencies
 
-Run
+To install the required dependencies with vcpkg, run:
 
     vcpkg install freetype giflib gtest libjpeg-turbo libpng libwebp pixman tinyxml zlib --triplet x64-windows
     
- Beware: `--triplet x64-windows` is only necessary for a 64 architecture
- build.
+Beware: `--triplet x64-windows` is only necessary for a 64 architecture
+build.
 
-After that you have to choose the back-end:
+### Mac OS X dependencies
 
-1. If you choose the Allegro back-end, you can jump directly to the
-   [Compiling](#compiling) section.
+On OS X you will need Mac OS X 10.11 SDK and Xcode 7.3 (older versions
+might work).
 
-2. If you choose the Skia back-end, you will need to
-   [compile Skia](#skia-on-windows) before and then continue in the
-   [Compiling](#compiling) section. Remember to check the
-   [Windows details](#windows-details) section to know how to call
-   `cmake` correctly.
+## Compiling
 
-## Mac OS X dependencies
+To compile LibreSprite, run the following commands:
 
-On OS X you will need Mac OS X 10.11 SDK and Xcode 7.3 (maybe older
-versions might work).
+    cd LibreSprite
+    mkdir build
+    cd build
 
-Also, you must compile [Skia](#skia-on-mac-os-x) before starting with
-the [compilation](#compiling).
+Then following the platform-specific instructions for compiling below.
 
-## Linux dependencies
+After you've finished compiling, the executable file will be placed in the
+`build/bin` directory. If you want to build a fresh copy of LibreSprite, remove
+the `build` directory and recompile.
 
-Debian/Ubuntu:
+Once you've finished compiling, you can install LibreSprite with the following
+command:
 
-    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+    ninja install
 
-The `libxcursor-dev` package is needed to
-[hide the hardware cursor](https://github.com/aseprite/aseprite/issues/913).
```

Comment:
I'd keep that. I guess if David felt the need to specify it, some people wondered why that library was there

Reply:
Right, but this doesn't really belong in the Installation instructions. It could probably go in some other document about LibreSprite's design.

Reply:
Mm... I smell mess scattered around the place. In parenthesis?

Reply:
Note this might also become inaccurate if we use SDL for this instead.

Reply:
I mean, we need to clean it in the future anyway